src/sentry/replays/query.py

```diff
@@ -582,6 +638,33 @@ def _activity_score():
 }
 
 
+TAG_QUERY_ALIAS_COLUMN_MAP = {
+    "replay_type": Column("replay_type"),
+    "platform": Column("platform"),
+    "environment": Column("environment"),
+    "release": Column("release"),
+    "dist": Column("dist"),
+    "url": Function("arrayJoin", parameters=[Column("urls")]),
+    "user.id": Column("user_id"),
+    "user.username": Column("user_name"),
+    "user.email": Column("user_email"),
+    "user.ip": Function(
+        "IPv4NumToString",
+        parameters=[Column("ip_address_v4")],
+    ),
+    "sdk.name": Column("sdk_name"),
+    "sdk.version": Column("sdk_version"),
+    "os.name": Column("os_name"),
+    "os.version": Column("os_version"),
+    "browser.name": Column("browser_name"),
+    "browser.version": Column("browser_version"),
+    "device.name": Column("device_name"),
+    "device.brand": Column("device_brand"),
+    "device.family": Column("device_family"),
+    "device.model": Column("device_model"),
```

Comment:
```suggestion
    "device.model_id": Column("device_model"),
```

`model_id` is the renamed field. it's a worse name, but matches with events table.

Reply:
thanks for the catch!